modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -2159,12 +2159,15 @@ export class Wallet implements IWallet {
     // this could return 100 build transactions
     const buildResponse = (await this.bitgo
       .post(this.baseCoin.url('/wallet/' + this.id() + '/consolidateAccount/build'))
-      .send(whitelistedParams)
-      .result()) as any;
+      .send(whitelistedParams)) as any;
 
-    // we need to step over each prebuild now - should be in an array in the body
     const consolidations: PrebuildTransactionResult[] = [];
-    for (const consolidateAccountBuild of buildResponse) {
+    if (buildResponse.status === 204) {
```

Comment:
can you provide comments for this behavior?

Reply:
Oh, the endpoint documentation will be updated on [this PR](https://github.com/BitGo/bitgo-microservices/pull/16613/files) but also I just added a little comment here as you suggested, let me know if it is ok this